deployments/fargate/signalfx-agent-fargate-task.json

```diff
@@ -0,0 +1,46 @@
+{
+    "containerDefinitions": [
+        {
+            "name": "redis",
+            "image": "redis:latest",
+            "essential": true,
+            "dockerLabels": {
+                "agent.signalfx.com.port.6379": "true",
+                "agent.signalfx.com.config.6379.intervalSeconds": "1"
+            }
+        },
+        {
+            "entryPoint": [
+                "bash",
+                "-c"
+            ],
+            "portMappings": [],
+            "command": [
+                "curl --fail $CONFIG_URL > /etc/signalfx/agent.yaml && exec /bin/signalfx-agent"
+            ],
+            "environment": [
+                {
+                    "name": "ACCESS_TOKEN",
+                    "value": "MY_ACCESS_TOKEN"
+                },
+                {
+                    "name": "CONFIG_URL",
+                    "value": "https://raw.githubusercontent.com/signalfx/signalfx-agent/master/deployments/ecs/agent.yaml"
+                }
+            ],
+            "dockerLabels": {
+                "app": "signalfx-agent"
+            },
+            "name": "signalfx-agent",
+            "image": "quay.io/signalfx/signalfx-agent:4.0.2"
```

Comment:
Make sure this line gets updated by the script in `scripts/update-deployment-versions`